packages/@chronobank/auth/actions.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Copyright 2017–2018, LaborX PTY
+ * Licensed under the AGPL Version 3 license.
+ *
+ * @flow
+ */
+
+// import * as AuthActionTypes from './constants'
+import { createActions } from 'redux-actions'
```

Comment:
What reasons of using 'redux-actions' package?
Maybe better to use straightforward way — simple pure functions?